extensions/vscode/snippets/freezed_bloc.json

```
@@ -0,0 +1,26 @@
+{
+  "create a state in a freezed class": {
+    "prefix": "fstate",
+    "body": "const factory ${TM_FILENAME_BASE/(.*)/${1:/pascalcase}/g}.${1:stateName}($2) = _${1/(.*)/${1:/capitalize}/};\n$3"
+  },
+  "create a event in a freezed class": {
+    "prefix": "fevent",
+    "body": "const factory ${TM_FILENAME_BASE/(.*)/${1:/pascalcase}/g}.${1:eventName}($2) = _${1/(.*)/${1:/capitalize}/};\n$3"
+  },
+  "create a map event to state function with freeze.when function": {
+    "prefix": "fmapToState",
```

Can we change this prefix to something more related to the purpose?
This snippet does not use `map`, and it is too close to the one below.
I would propose `feventToState` for this one, and `fmapEventToState` for the one below that uses `map`.
```suggestion
    "prefix": "feventToState",
```